notes

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+require('es6-promise').polyfill();
+require('isomorphic-fetch');
+
+// const emptyAnswer = { title: '', score: -1 };
+// const emptyQuestion = { title: '', answers: [emptyAnswer], id: 1 };
+// const emptyQuiz = { title: '', questions: [emptyQuestion] };
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      quiz: null
+    };
+  }
+
+  componentDidMount() {
+    this.retrieveQuestions();
+  }
+
+  retrieveQuestions() {
+    const questionsUrl = () => `http://localhost:3001/quizzes/1`;
```

Comment:
(In the future, don't commit this to a repo. Just save it as a file on your Desktop or something.)